patches/common/v8/.patches.yaml

```diff
@@ -1,5 +1,16 @@
 repo: src/v8
 patches:
+-
+  owners: codebytere
+  file: add_realloc.patch
+  description: |
+    Blink overrides ArrayBuffer's allocator with its own one, while Node simply
+    uses malloc and free, so we need to use v8's allocator in Node. As part of the
+    10.6.0 upgrade, we needed to make SerializerDelegate accept an allocator
+    argument in its constructor, and override ReallocateBufferMemory and
+    FreeBufferMemory to use the allocator. We cannoy simply allocate and then memcpy
+    when we override ReallocateBufferMemory, so we therefore need to implement
```

Comment:
Why not? That is, why not implement `ReallocBufferMemory` in terms of 
existing methods `AllocateUninitialized` and `Free`?

```cpp
ReallocateBufferMemory(src, size) {
  void * tgt = AllocateUninitialized(size);
  if (tgt != NULL) {
    memcpy(tgt, src, size);
    Free(src);
  }
  return tgt;
}

Reply:
@ckerr:
`ReallocateBufferMemory` has the following signature:
```void* ReallocateBufferMemory(void* old_buffer,
                               size_t size,
                               size_t* actual_size)
```
and we don't have access to the old buffer's size in order to use `memcpy` as it is plain memory